py/torch_migraphx/fx/converters/acc_ops_converters.py

```diff
@@ -1998,6 +1998,52 @@ def acc_ops_isinf(mgx_module, node, args, kwargs):
         mgx_module.add_instruction(migraphx.op('isinf'), [inp.instr_ref]))
 
 
+@migraphx_converter(acc_ops.any)
```

Comment:
Once #158 is merged, you can update this and the other converters here to
```suggestion
@migraphx_converter(acc_ops.any, min_migraphx_ver="2.11.0")
```